examples/universal-cnf/vppagent/pkg/ucnf/ucnf_nse.go

```diff
@@ -1,15 +1,94 @@
 package ucnf
 
 import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"time"
+
 	"github.com/danielvladco/k8s-vnet/pkg/nseconfig"
 	"github.com/davecgh/go-spew/spew"
-	"github.com/tiswanso/examples/examples/universal-cnf/vppagent/pkg/config"
+	"github.com/gofrs/uuid"
 	"github.com/networkservicemesh/networkservicemesh/sdk/common"
 	"github.com/sirupsen/logrus"
+	"github.com/tiswanso/examples/api/ipam/ipprovider"
+	"github.com/tiswanso/examples/examples/universal-cnf/vppagent/pkg/config"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/grpc"
 	"gopkg.in/yaml.v2"
-	"os"
 )
 
+type ipamService struct {
+	IpamAllocator     ipprovider.AllocatorClient
+	registeredSubnets chan *ipprovider.Subnet
+	ctx               *context.Context
```

Comment:
Don't use a pointer to `context.Context`. Simply use context.

Reply:
done